client/reader/discover/controller.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import ReactDom from 'react-dom';
+import moment from 'moment';
+
+/**
+ * Internal dependencies
+ */
+import analytics from 'lib/analytics';
+import config from 'config';
+import route from 'lib/route';
+import trackScrollPage from 'lib/track-scroll-page';
```

Comment:
this is different from the reader's trackScrollPage. It doesn't bump the reader view mc stat.